docker/candlepin-base/cp-test

```diff
@@ -231,4 +241,12 @@ if [ "$RSPEC" == "1" ]; then
     buildr "${BUILDR_ARGS}" > /tmp/teepipe 2>&1
 fi
 
+if [ ! -z "$ARBITRARY" ]; then
+    cd "$CP_HOME"
+    echo "Running buildr $ARBITRARY..."
+    tee /var/log/candlepin/buildr.log < /tmp/teepipe &
+
+    buildr "${ARBITRARY}" > /tmp/teepipe 2>&1
```

Comment:
If we change this line to ```buildr $ARBITRARY > ...```, we can issue compound commands via -b. Quick way to test this is with ```cp-test -b "clean test"```. As written now, it quotes the 'clean test' and builder attempts to run it as one task; changing it to not quote $ARBITRARY acts as a more accurate passthrough.